source/main/utils/detectSystemLocale.js

```diff
@@ -9,10 +9,7 @@ export const detectSystemLocale = (): string => {
   if (systemLocale === 'ja') {
     return LOCALES.japanese;
   }
-  if (systemLocale === 'en-US') {
-    return LOCALES.english;
-  }
-  return systemLocale;
+  return LOCALES.english;
 };
 
 export const detectSystemDateLocale = (): string => {
```

Comment:
@DeeJayElly now both methods return the same value. What about `detectSystemLocale`?
Don't we want to have this part in the `detectSystemDateLocale ` method?
![Screenshot 2019-08-19 at 15 05 31](https://user-images.githubusercontent.com/376611/63267647-d40ac480-c292-11e9-8f62-8e36d39d61ab.png)

Reply:
They dont return the same thing, look at it from this perspective:

export const detectSystemLocale = (): string => {
  const systemLocale = app.getLocale();
  Logger.info('Detected system locale', { systemLocale });
  if (systemLocale === 'ja') {
    return LOCALES.japanese;
  }
  return LOCALES.english;
};

export const detectSystemDateLocale = (): string => {
  const systemDateLocale = app.getLocale();
  return systemDateLocale;
};


detectSystemLocale() in Japanese case will return 'ja-JA'
while detectSystemDateLocale() will return 'ja' due to https://electronjs.org/docs/api/locales

Reply:
Thanks @DeeJayElly - got it now! It's also not returning English as a fallback which is good.
However, are we sure the "ja" value (and other locales) will work in moment.js?

Reply:
> Thanks @DeeJayElly - got it now! It's also not returning English as a fallback which is good.
> However, are we sure the "ja" value (and other locales) will work in moment.js?

Sure it will work, checkout : https://momentjs.com/docs/#/i18n/

Reply:
Awesome 💯